standalone/source/main.cpp

```diff
@@ -6,15 +6,20 @@
 #include <string>
 #include <unordered_map>
 
-const std::unordered_map<std::string, greeter::LanguageCode> languages{
-    {"en", greeter::LanguageCode::EN},
-    {"de", greeter::LanguageCode::DE},
-    {"es", greeter::LanguageCode::ES},
-    {"fr", greeter::LanguageCode::FR},
-};
-
+// NOLINTNEXTLINE(modernize-use-trailing-return-type)
 int main(int argc, char** argv) {
-  cxxopts::Options options(argv[0], "A program to welcome the world!");
+  // prevent warning: initialization of 'languages' with static storage duration may throw an
+  // exception that cannot be caught [cert-err58-cpp]
+  const std::unordered_map<std::string, greeter::LanguageCode> languages{
+      {"en", greeter::LanguageCode::EN},
+      {"de", greeter::LanguageCode::DE},
+      {"es", greeter::LanguageCode::ES},
+      {"fr", greeter::LanguageCode::FR},
+  };
+
+  // prevent warning: do not use pointer arithmetic
+  // [cppcoreguidelines-pro-bounds-pointer-arithmetic]
+  cxxopts::Options options(*argv, "A program to welcome the world!");
```

Comment:
Do we actually need all those comments? That's useful only for certain `clang-tidy` configurations, correct?
Imo clang-tidy is nice, but not a necessary feature for the starter, so it can be removed.

Reply:
no, that was only for your
I will remove this